__tests__/app.test.js

```diff
@@ -65,23 +65,78 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe.only("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
```

Comment:
When we are iterating through a results array here, if the array is empty we will auto pass the tests below. It is essential when iterating through an array in a test like this, that we first check the array has length to avoid this.